<?xml version = "1.0"?>

<launch>
    <arg name = "servo1_angle" default = "1.0" />
    <arg name = "servo2_angle" default = "-1.0" />

    <node name = "servo_control" pkg = "my_dynamixel_tutorial" type = "servo_control_node" output = "screen">
        <param name = "servo1_angle" value = "$(arg servo1_angle)" />
        <param name = "servo2_angle" value = "$(arg servo2_angle)" />
        <param name = "L1" value = "0.22360679" />
        <param name = "L2" value = "0.471699" />
    </node>

</launch>